opm/input/eclipse/share/keywords/900_OPM/R/REGION_PROBE_OPM

```diff
@@ -37,7 +37,8 @@
     "RMUPT",
     "RWCD",
     "RWIPG",
-    "RWIPL"
+    "RWIPL",
+    "RWMAIP"
```

Comment:
Region level summary vectors can't have more than five characters.  I see that you introduce a way to identify that case, but really, it's a **fundamental** limit that these vectors have at most five characters in their base names so you need to do something else.

Reply:
Using `AM` instead of `WMA` then.